lib/web/fetch/request.js

```diff
@@ -400,16 +400,6 @@ class Request {
 
     // 29. If signal is not null, then make this’s signal follow signal.
     if (signal != null) {
-      if (
```

Comment:
this is no longer needed because we are actually validating signal in webidl now